documentation/asciidoc/computers/raspberry-pi/boot-http.adoc

```diff
@@ -40,6 +40,79 @@ rpi-eeprom-digest -i pieeprom.upd -o pieeprom.sig
 rpi-eeprom-digest -i boot.img -o boot.sig -k myprivkey.pem
 ----
 
+=== Certificates
+
+For security, Network Install uses HTTPS to download OS images from the Raspberry Pi website. This feature uses our own CA root included in the bootloader to verify the host.
+
+You can add your own custom CA certificate to the EEPROM to securely download images from your own website. Use the `--cacertder` option of the `rpi-eeprom-config` tool to add the DER-encoded certificate.
+
+To ensure that the certificate is not modified, place a hash of the certificate in the EEPROM config settings.
+
+Run the following command to generate a DER-encoded certificate:
+
+[source,console]
+----
+$ openssl x509 -in your_ca_root_cert.pem -out cert.der -outform DER
+----
+
+Then, run the following command to generate a SHA-256 hash of the certificate:
+
+[source,console]
+----
+$ sha256sum cert.der
+----
+
+You should see output similar to the following:
+
+----
+701bd97f67b0f5483a9734e6e5cf72f9a123407b346088638f597878563193fc  cert.der
+----
+
+Next, update `boot.conf` to include the hash of the certificate:
+
+[source,console]
+----
+$ sudo rpi-eeprom-config --edit
+----
+
+Configure the following settings in the `[gpio8=0]` section, replacing:
+
+* `<your_website>` with your website, e.g. `raspberrypi.com`
```

Comment:
Perhaps the "e.g. `raspberrypi.com`" should be changed to "e.g. `raspberrypi.com`" so that it matches the `yourserver.org` in the "You can check the boot output for the HTTPS URL" section?
Similarly, perhaps the `path/to/image` in the next line should be updated to match the `path/to/files` in the example boot output?

Reply:
I think this gets the message aross?

Reply:
```suggestion
* `<your_website>` with xref:raspberry-pi.adoc#HTTP_HOST[your website], e.g. `yourserver.org`
```